src/main/resources/template/select.vm

```diff
@@ -0,0 +1,104 @@
+#* @vtlvariable name="action" type="com.atlassian.shele.shelePlugin.webwork.SelectAction" *#
+
+
+
+#set($context=$action.servletContext)
+<html>
+<head>
+  <meta name="decorator" content="atl.admin"/>
+  <meta name="admin.active.section" content="admin_plugins_menu/my-section"/>
+  <meta name="admin.active.tab" content="my-next-item"/>
+  <script src="https://connect-cdn.atl-paas.net/all.js"></script>
+  <title>$i18n.getText("my-web-item.name.title.loads")</title>
+</head>
+<body>
+<form class="aui" action=$baseurl/rest/rest-issue/1.0/api/issue method="post" id="form-1">
+  <select class="multi-select" size="4" multiple="multiple" id="select2-example" multiple >
+    <label>Projects</label>
+      #foreach($pro in $context.getAttribute("prod"))
+          #if($pro)
+          <option id="opt" value=$pro.getId()>
+                $pro.getKey()
+          </option>
+          #end
+      #end
+  </select>
+  <div>
+
+    <select class="multi-select" size="4" multiple="multiple" id="event" name="issue-example">
+      <label>Events</label>
+    #foreach($list in $context.getAttribute("event"))
+      #if($list)
+      <option id="test" value=$list.getId()>$list.getName()
+      </option>
+          #end
+        #end
+    </select>
+  </div>
+  <div class="bt-example">
+    <button class="aui-button aui-button-primary" id="showAction" class="text-left" style="display: block; margin-right: auto;">$i18n.getText("my-web-item.name.button")</button>
+  </div>
+</form>
+<div class="error" id="valid">
+  <ul>
+    <li><span class="aui-icon aui-icon-small aui-iconfont-error aui-icon-notification"></span>Some
+      field has a problem.
+    </li>
+  </ul>
+</div>
+<script type="text/javascript">
+  AJS.toInit(function (){
+
+    AJS.$('#valid').hide();
+    AJS.$('#form-1').submit(function (event) {
+    AJS.$('#showAction').hide();
+    event.preventDefault();
+    var attr = $('#test').attr('value');
+    console.log(attr);
+    var attr2 = $('#test2').attr('value2');
+    console.log(attr2);
+      var attr3 = $('#test3').attr('value3');
+      var attr4 = $('#test4').attr('value4');
+      let arrs=[attr,attr2,attr3,attr4];
+    var value = $('#form-1').attr('action');
+    var host = value.valueOf();
+
+
+
+      let sk=$('#select2-example').val();
+        if (sk===null)
+         AJS.$('#valid').show();
+      //  window.location.reload();
+          var g = $('#event').val();
+            if (g ===null)
+              AJS.$('#valid').show();
+
+
+    var a = {
+      project: sk,
+      eventTypeIds:g
+    }
+    console.log(a);
+
+  var cancel =  jQuery.ajax({
+      type: "POST",
+      data: JSON.stringify(a),
+      url: host,
```

Comment:
you can set up the host here directly without any variables, it has constant value